lib/file_system_helper.sh

```diff
@@ -5,7 +5,7 @@
 get_distribution_docroot() {
     local docroot="web"
 
-    if [[ ${DRUPAL_TRAVIS_DISTRIBUTION} = "thunder" ]]; then
+    if [[ ${DRUPAL_TRAVIS_COMPOSER_PROJECT} =~ ^thunder/thunder-project.* ]]; then
```

Comment:
In future we could maybe use 
`
        "drupal-scaffold": {
            "locations": {
                "web-root": "web/"
            }
        },
`